src/content/syllabus.md

```diff
@@ -160,6 +160,21 @@ On time: no bonus
 
 Only your last submission will be graded. 
 
+### Generative AI (e.g. ChatGPT)
+Use of generative AI on problem sets is encouraged after attempting the problem sets by yourself.
```

Comment:
I think we need to be more clear what we mean by this - and is it intended to discourage/disallow using LLMs when students are stuck and don't know how to start?

I think we want to first get the intent of the rules across - we want students to use resources (including other people and AI tools) to aid their learning and improve their experience, but not as a substitute for learning. I would just make the policy that students need to document the resources they use, and if they use AI tools to explain how and provide links to the transcripts.

Reply:
Thoughts on something like this?

## Generative AI
Generative AI can be a useful tool in various aspects of many parts of life including Computer Science. However, part of the reason to get a Computer Science degree is to teach problem solving, and often generative AI is used to bypass the learning process. One of the goals of our class is to teach you how to use generative AI to augment your learning rather as a substitute for learning. 

Encouraged usages of GenAI:
...
Discouraged usages of GenAI:
...

## Documenting outside sources
When working on problem sets all outside resources *should be documented* in a "Outside Resources Used" section at the end of each problem set. Within the class resources which do not need to be documented are: lecture material/slides, discussions with DMT1 course TAs, material from the course textbook. Outside resources include, but are not limited to: generative AI tools (e.g. ChatGPT, claude, copilot), results from Google searches (e.g. stackexchange), and discussions with classmates (TODO should this be considered outside of the material)?. 

*Examples:*
GenAI:
```
ChatGPT -
Me: What are examples of a powerset?
ChatGPT: A powerset is the set of all subsets of a given set, including the empty set and the set itself. Here are examples to illustrate the concept:

Example 1: Simple Set
Let 𝐴 = { 1, 2 }
A={1,2}.

The powerset of 𝐴
A is: 𝑃(𝐴) = {∅, {1}, {2}, {1,2}}
P(A)={∅,{1},{2},{1,2}}
...
```
Google searches:
```
https://math.stackexchange.com/questions/3226497/power-set-definitions
```
Discussions with classmates:
```
I discussed how powersets work and some examples of powersets with Dave Evans
```
```
Dave Evans and I worked together on structuring our proof
```

Reply:
I like this, but think we should just put it on the first assignment, rather than the syllabus. If it is actually on the assignment, people will read it at the right time. We can do some demos using LLMs in class.